Xamarin.Forms.Platform.MacOS/Renderers/BoxViewRenderer.cs

```diff
@@ -35,5 +39,16 @@ protected override void SetBackgroundColor(Color color)
 			Control.WantsLayer = true;
 			Control.Layer.BackgroundColor = color.ToCGColor();
 		}
+
+		private void SetCornerRadius()
+		{
+			if (Element == null)
+				return;
+
+			var elementCornerRadius = Element.CornerRadius;
+
+			Layer.MasksToBounds = true;
+			Layer.CornerRadius = (float)elementCornerRadius.TopLeft;
```

Comment:
Does this ignore all of the other CornerRadius values?

Reply:
Updated. An **NSBezierPath** is used to drawing and set each corner.